src/mod_config.lua

```diff
@@ -19,37 +19,136 @@ author: G.A. vd. Hoorn
 
 mod_config =
 {
-  controlled_vehicle = {
-    base_link_frame_id = "case_ih_7210_pro_9"
-  },
-  laser_scan = {
-    num_rays = 128,
-    num_layers = 5,
-    angle_min = -math.pi,
-    angle_max = math.pi,
-    range_min = 0.1,
-    range_max = 30,
-    ignore_terrain = true,
-    inter_layer_distance = 0.1,
-    -- apply a transform to first laser scan frame
-    laser_transform = {
-      translation = {
-          x = 0.0,
-          y = 0.0,
-          z = 0.0
+  control_only_active_one = false,
+  vehicle = {
+    case_ih_7210_pro = {
```

Comment:
the names here are the sanitized (replacing whitespace with underscores and uppercase with lowercase) full name. and if a vehicle is not in the config list then there will be errors as the LaserScanner table will be nil for that vehicle.

Reply:
> if a vehicle is not in the config list then there will be errors as the LaserScanner table will be nil for that vehicle.

ok, then it might be good to either define some 'default' settings which get used in that case, or return an error and stop initialising a laser scanner for those vehicles.